clients/cli/src/main.rs

```diff
@@ -112,6 +113,12 @@ enum Lockbook {
         path: String,
     },
 
+    /// Print the file tree with a given file as the head
+    PrintTree {
+        /// Absolute path of a file !@#$%^&*I don't know what this is:(lockbook list-docs)
+        path: String,
```

Comment:
I don't think you have to worry about only printing out the subtree, just print out the whole business, simpler that way too.